.github/workflows/rust.yml

```diff
@@ -8,25 +8,31 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  RUSTFLAGS: "-Dwarnings"
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v3
-    - name: Build and Check style
-      run: |
-        cargo build --verbose
-        cargo fmt --all -- --check
+      - uses: actions/checkout@v3
```

Comment:
Avoid unnecessary diffs and make sure you do minimal changes.

Reply:
> Avoid unnecessary diffs 

Hm?

> make sure you do minimal changes.

It seemed to be a minor change. Well 
![image](https://github.com/minio/minio-rs/assets/17514622/eddca60e-04bb-4e36-8a4d-83bd8c7f74ee)

Reply:
Ehs. I may be a little tired…

Reply:
We do not need `yamllint` . Please add what is really needed here.